spark/src/test/java/com/netflix/iceberg/spark/source/TestTables.java

```diff
@@ -73,7 +72,8 @@ private TestTable(TestTableOperations ops, String name) {
       this.ops = ops;
     }
 
-    TestTableOperations ops() {
+    @Override
+    public TestTableOperations operations() {
```

Comment:
Why was this rename needed?

Reply:
The previous version didn't actually override the superclass's method.

Reply:
Probably because we didn't expose `TableOperations` at first. Now that `FileIO` will be public, we can probably remove `HasTableOperations` in a follow-up.